source/mongodump.txt

```diff
@@ -264,6 +264,13 @@ Behavior
    the :dbtools:`--archive </mongodump/#std-option-mongodump.--archive>` 
    option.  
 
+Using ``mongodump`` Without an ``authSource``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When :urioption:`authSource` is not specified in the MongoDB URI, the database name provided 
+with :option:`--db` is used to authenticate your ``mongodump`` session
+and indicate the database being dumped.
```

Comment:
[optional]

Do you think it'd be worth adding a clarifying sentence like:

"To authenticate to a different database than the one being dumped, you must specify the ``authSource`` option."

Reply:
I was speaking with the PM on this, I think I will add an additional example to the page, and I will add a ref to this blurb to help clarify the behavior with a concrete code based example.